integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/util/ApictlUtils.java

```diff
@@ -119,6 +121,24 @@ public static void createProject(String openApiFile, String apiProjectName) thro
         createProject(openApiFile, apiProjectName, null, null, null);
     }
 
+    /**
+     * Create an API project - To be used before deploying an API via the apictl deploy command
```

Comment:
Shall we mention why this method is different so that we wouldn't have to go back and compare?